ui/pages/destinations/index.js

```diff
@@ -290,13 +290,33 @@ const columns = [
   {
     Header: 'Kind',
     accessor: v => v,
-    width: '33%',
+    width: '25%',
     Cell: ({ value }) => (
       <span className='rounded bg-gray-800 px-2 py-0.5 text-gray-400'>
         {value.kind}
       </span>
     ),
   },
+  {
+    Header: 'Status',
```

Comment:
This doesn't necessarily need a column. An alternative may be to add it as an icon in the `Last Seen` column @FSHA